test/functional/wallet_change_address.py

```diff
@@ -62,5 +71,42 @@ def run_test(self):
                 # find the change output and ensure that expected change index was used
                 self.assert_change_index(self.nodes[n], tx, i)
 
+        # Drain wallets
+        for i in range(1, 3):
+            node = self.nodes[i]
+            amount = node.getbalance()
+            node.sendtoaddress(self.nodes[0].getnewaddress(), amount, subtractfeefromamount=True)
```

Comment:
In 80ad44d10ca0831869a5e50c36e07e7ff10902ec "wallet: do not shadow change_pos in CreateTransaction()"

Use `sendall` instead of `sendtoaddress` with `subtractfeefromamount`.

Or just make a new wallet for this test.

Reply:
done, thanks